client/src/app/employee/employee-detail/employee-detail.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Employee } from './state/employee-detail.model';
+
+@Component({
+  selector: 'app-employee-detail',
+  templateUrl: './employee-detail.component.html',
+  styleUrls: ['./employee-detail.component.scss'],
+})
+export class EmployeeDetailComponent implements OnInit {
+  @Input()
+  employee: Observable<Employee>;
```

Comment:
This should probably be `@Input() employee: Employee` and the `| async` should happen in the parent component passing the state in.

Reply:
So Justin taught me something new today, I never realized you could async pipe an input like `[employee]="this.employee | async"`. So now this makes more sense on how to have the parent component manage gathering all the data, handle any observables, and then push that data into "dumb" child components.